src/main/kotlin/no/nav/syfo/service/SenderFacade.kt

```diff
@@ -70,12 +74,13 @@ class SenderFacade(
                 feilmelding = e.message,
                 journalpostId = null,
                 brukernotifikasjonerMeldingType = null,
+                isForcedLetter = false,
             )
         }
     }
 
     fun sendTilBrukernotifikasjoner(
-        uuid: String,
+        uuid: String, // aktivitetskravUuid
```

Comment:
Aktivitetskrav? Er ikke dette både til dialogmøter, oppfølgingsplan osv?

Reply:
Jeg skal fjerne det, lagt det for å tydeligjøre det i oppgavens kontekst